package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "prepublish": "npm run build",
     "test": "npm run lint && npm run mocha",
-    "mocha": "mocha --require babel-core/register test/**/*.js",
+    "mocha": "mocha --require babel-core/register ./test/**/*.js ./test/*.js",
```

Comment:
i'm not sure this change is necessary, both my machine and Travis have no problem running the tests in their current setup